src/main/webapp/app/entities/programming-exercise/programming-exercise.component.html

```diff
@@ -163,6 +196,8 @@ <h4>
                                     [jhiFeatureToggle]="FeatureToggle.PROGRAMMING_EXERCISES"
                                     type="submit"
                                     *ngIf="programmingExercise.isAtLeastInstructor && programmingExercise.templateParticipation"
+                                    jhiIdeFilter
+                                    [showInIDE]="false"
```

Comment:
showInOrion?

Reply:
This will also be changed in the next PR: #1155